web-wallet/src/routes/(app)/dashboard/+page.svelte

```diff
@@ -52,20 +50,22 @@
   }
 
   /** @type {(descriptors: ContractDescriptor[]) => ContractDescriptor[]} */
-  const getEnabledContracts = filterWith(hasKeyValue("disabled", false));
+  const getEnabledContracts = filterWith(hasKeyValue("enabled", true));
```

Comment:
Do we get any benefit from using `filterWith`?

Why not just:

```
const enabledContracts = filter(
  contractDescriptors,
  hasKeyValue("enabled", true)
);
```

cc: @ascartabelli

Reply:
Only advantage would be reusability: if the function is called multiple times or if we want to move it out and put it, for example, in a `$lib/contracts` folder so that the consumer doesn't care about the logic of what makes a contract enabled.

If there is no such case we can just use filter and, if the array of contracts is supposed to be small, we can also ditch Lamb entirely and use the native filter method of arrays.